src/vsr/replica.zig

```diff
@@ -3231,11 +3745,34 @@ pub fn Replica(
                 return;
             }
 
+            if (self.op < config.journal_slot_count) {
+                // The op is known, and this is the first WAL cycle.
+                // Therefore, any faulty ops to the right of `replica.op` are corrupt reserved
+                // entries from the initial format.
+                var op: usize = self.op + 1;
+                while (op < config.journal_slot_count) : (op += 1) {
+                    const slot = self.journal.slot_for_op(op);
+                    assert(slot.index == op);
+
+                    if (self.journal.faulty.bit(slot)) {
+                        assert(self.journal.headers[op].command == .reserved);
+                        self.journal.dirty.clear(slot);
+                        self.journal.faulty.clear(slot);
+                        log.debug("{}: repair_prepares: op={} (op known, first cycle)", .{
+                            self.replica,
+                            op,
+                        });
+                    }
+                }
+            }
```

Comment:
> Is this required for the rest of the protocol to make progress?

Initially it was required to enable the initial startup in the VOPR with a corrupt WAL. But I've updated the VOPR since then so it shouldn't be necessary anymore.